client/my-sites/customer-home/components/home-survey/index.jsx

```diff
@@ -0,0 +1,50 @@
+import { useIsEnglishLocale } from '@automattic/i18n-utils';
+import ReactDOM from 'react-dom';
+import SurveyModal from 'calypso/components/survey-modal';
+import { useSelector } from 'calypso/state';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import homeSurveyImage from './assets/images/home-survey.png';
+
+const HomeSurvey = () => {
+	const site = useSelector( getSelectedSite );
+	const isEnglishLocale = useIsEnglishLocale();
+	const isSiteLaunched = site?.launch_status === 'launched' || false;
+	const isSiteCreatedInLast3Days =
+		site?.options?.created_at &&
+		new Date( site.options.created_at ) > new Date( Date.now() - 3 * 24 * 60 * 60 * 1000 );
+	const isLastActivityWithin2Days =
+		site?.options?.updated_at &&
+		new Date( site.options.updated_at ) > new Date( Date.now() - 2 * 24 * 60 * 60 * 1000 );
+	const isTwentyTwentyFiveTheme = site?.options?.theme_slug === 'pub/twentytwentyfive' || false; // The old default theme.
+	const isRetrospectTheme = site?.options?.theme_slug === 'pub/retrospect' || false; // The new default theme. See 185017-ghe-Automattic/wpcom.
+
+	const isEligibleForSurvey =
+		isEnglishLocale &&
+		isSiteLaunched &&
+		isSiteCreatedInLast3Days &&
+		isLastActivityWithin2Days &&
+		! isTwentyTwentyFiveTheme &&
+		! isRetrospectTheme;
+
+	if ( ! isEligibleForSurvey ) {
+		return null;
+	}
+
+	return ReactDOM.createPortal(
+		<SurveyModal
+			name="survey-home"
+			eventName="calypso_survey_home"
+			title="Shape the Future of WordPress.com"
+			description="Are you interested in participating in a research interview and earning a $50 credit on your WordPress.com account? We are looking for feedback!"
```

Comment:
I'm not sure about the future plans, but if we plan to turn it ON for other languages, we should translate this.
The same for the dismiss and confirm texts.

Reply:
I think we can wrap it with the translation function right now, so if we plan to extend it to non-EN locales, it's ready to be used.

Reply:
Agree, it doesn't hurt to have it. Updated in https://github.com/Automattic/wp-calypso/pull/104161/commits/fc640df60b221b8e9721c4d6b78f30b8ff24e8d9. Thanks!